kotlinx-coroutines-test/common/src/TestBuilders.kt

```diff
@@ -578,8 +578,8 @@ internal expect fun dumpCoroutines()
 @JvmName("runTest\$default")
 @Suppress("DEPRECATION", "UNUSED_PARAMETER")
 public fun TestScope.runTestLegacy(
-    dispatchTimeoutMs: Long?,
+    dispatchTimeoutMs: Long,
     testBody: suspend TestScope.() -> Unit,
-    unused1: Int?,
+    unused1: Int,
     unused2: Any?,
-): TestResult = runTest(dispatchTimeoutMs ?: 60_000, testBody)
+): TestResult = runTest(dispatchTimeoutMs, testBody)
```

Comment:
```suggestion
): TestResult = runTest(if (unused1 and 1 != 0) DEFAULT_DISPATCH_TIMEOUT_MS else dispatchTimeoutMs, testBody)
```

Reply:
I think there's no need: if somebody did actually provide `dispatchTimeoutMs`, the `$default` overload wouldn't be chosen.

Reply:
Isn't that the wrong way around? We're in this function because they didn't supply all the parameters, therefore we need to fill them in.

Reply:
Oh, yes, you're right, it *is* the other way around. Let me think about it.

Reply:
You're fully correct, by decompiling this method in 1.6.4, we get
```java
   // $FF: synthetic method
   public static void runTest$default(TestScope var0, long var1, Function2 var3, int var4, Object var5) {
      if ((var4 & 1) != 0) {
         var1 = 60000L;
      }

      TestBuildersKt.runTest(var0, var1, var3);
   }
```
Thanks for saving us from one more potential bugfix release!